### PR TITLE
made a common function to display and hide tooltip

### LINKED ISSUE #45: Code Reduction
currently to display tooltip and hide there are two functions, which we can achieve by only one function.

### PR DESCRIPTION
**Pull request by Binu42**
#### code Reduction
Fix #45 
currently to display tooltip and hide there are two functions, which we can achieve by only one function.

### CODE REVIEW

**Review by ankiiitraj — COMMENTED** [review]
LGTM! 🎉

File: src/lib/index.tsx [hunk 1]
```diff
@@ -79,12 +79,8 @@ const CopyMailTo = ({
...
   };
 
-  const displayTooltip = () => {
-    setShowTooltip(true);
-  };
-
-  const hideTooltip = () => {
-    setShowTooltip(false);
+  const toggleTooltip = () => {
+    setShowTooltip(prev=> !prev);
```
Comment (prateek3255) [inline src/lib/index.tsx:83]:
```suggestion
    setShowTooltip(prevTooltipState => !prevTooltipState);
```

**Comment by prateek3255** [comment]
Thanks for working on this @Binu42

**Comment by prateek3255** [comment]
@all-contributors please add binu42 for code

**Pull request merged**